### ISSUE TITLE
Option to disable Dark Mode

### ISSUE THREAD

**Issue by Dantasstic** [M2Team/NanaZip#446]
I don't like the look of Dark Mode for NanaZip, so I would like the ability to toggle it off. Can this be added?

**Comment by modz2014** [comment]
put your system in light mode

**Comment by modz2014** [comment]
but yer the background color is too dark and should 255,45,45

**Comment by Dantasstic** [comment]
🙄 I don't want my entire system in light mode, I want this particular app in light mode. Tons of other apps have toggles to follow the system settings or select light or dark. This is pretty standard.

**Comment by modz2014** [comment]
yes i know i will try and push something for it

**Comment by modz2014** [comment]
the project is all over the place now will take me some time to understand it all again

**Comment by Dantasstic** [comment]
> yes i know i will try and push something for it

That would be lovely :)

**Comment by eggbean** [comment]
> yes i know i will try and push something for it

I don't want dark mode in this software either. I can't see any `%APPDATA%` files for NanaZip, so please give information on which registry key the setting uses so I can automatically add it in setup scripts.

**Comment by VermillionFury007** [comment]
I would love this too!!

**Comment by notNSANE** [comment]
adding my overwhelming support for this feature.

Kinda ironic that this feature is useless for me in 95% of apps but when I actually could use it, it doesn't exist 😂

**Comment by KristoferYG** [comment]
yeah,  why the fuk there is no toggle for white mode.... who asked this dark mode?

**Comment by modz2014** [comment]
the annoying thing is you have to build the project without Visual Studio just to test it which is so annoying

**Comment by modz2014** [comment]
I've added a checkbox under the settings for now, but I still need to do more testing. This is because NanaZip's dark mode isn't applied dynamically while the application is running.

**Comment by modz2014** [comment]
@MouriNaruto cant seem to change the darkmode dynamically at runtime also reason why your using dpi

**Comment by MouriNaruto** [comment]
You should see the implementation in https://github.com/M2Team/NanaZip/blob/main/NanaZip.Frieren/NanaZip.Frieren.DarkMode.cpp

Kenji Mouri

**Comment by modz2014** [comment]
@MouriNaruto this is only happening when the system changes from light to darkmode while the application is running not when the button is checked and unchecked

**Comment by MouriNaruto** [comment]
> I've added a checkbox under the settings for now, but I still need to do more testing. This is because NanaZip's dark mode isn't applied dynamically while the application is running.

NanaZip won't accept the dark mode switching unless you change your system's dark/light theme setting or app initializing.

Kenji Mouri

**Comment by modz2014** [comment]
@MouriNaruto so you wanting the user to go to there system settings all the time to change from light to dark mode

**Comment by MouriNaruto** [comment]
> [@MouriNaruto](https://github.com/MouriNaruto) this is only happening when the system changes from light to darkmode while the application is running not when the button is checked and unchecked

Because I never consider about the disable switch for that. You need to add some logic for the dark mode engine.

Kenji Mouri

**Comment by modz2014** [comment]
@MouriNaruto ok i will scrap the idea then but we need to fix the darkmode being so dark but since your using dpi only good when hdr is off which you already know this

**Comment by k1tc3t** [comment]
> [@MouriNaruto](https://github.com/MouriNaruto) хорошо, тогда я откажусь от этой идеи, но нам нужно исправить тёмный режим, который слишком тёмный, но поскольку вы используете только разрешение в пикселях, это хорошо только при отключённом HDR, о чём вы уже знаете

so the dark theme wasn't fixed? It's too dark and there are no horizontal dividing lines :(
I also wish it was possible to force the light and dark themes to be changed.

PLEASE

**Comment by minus21grams** [comment]
Is this too hard to implement I wonder... 🤔

**Comment by DevX-Cipher** [comment]
@minus21grams i haven't done it sorry because developer didn't want to do it

**Comment by Furshia** [comment]
Still no update on this?

**Comment by RetroCro** [comment]
The fact that NanaZip can't do this one basic feature makes it worse than the app it's trying to replace.
